fcrepo-exts/migration-utils
# update to use caching

**JIRA Ticket**: https://fedora-repository.atlassian.net/browse/FCREPO-3693

**Depends on https://github.com/fcrepo/fcrepo-storage-ocfl/pull/42**

# What does this Pull Request do?

Configures the migration utils to use caches in the ocfl object sessions, which greatly improves the performance problems reported by Calvin.

# How should this be tested?

migration-utils should run almost as fast as Calvin originally reported in the pre-alpha build.

# Interested parties

@fcrepo/committers
